sv_pipeline/genome/load_data.py

```diff
@@ -0,0 +1,181 @@
+import argparse
+import os
+import time
+import hail as hl
+import logging
+
+from hail_scripts.v02.utils.elasticsearch_client import ElasticsearchClient
+
+from sv_pipeline.utils.common import get_sample_subset, get_sample_remap, get_es_index_name, CHROM_TO_XPOS_OFFSET
+from sv_pipeline.genome.utils.mapping_gene_ids import load_gencode, get_gene_id
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+WGS_SAMPLE_TYPE = 'WGS'
+BASIC_FIELDS = {
+        'contig': lambda rows: rows.locus.contig.replace('^chr', ''),
+        'sc': lambda rows: rows.info.AC[0],
+        'sf': lambda rows: rows.info.AF[0],
+        'sn': lambda rows: rows.info.AN,
+        'svType': lambda rows: hl.if_else(rows.alleles[1].startswith('<INS:'), 'INS', rows.alleles[1].replace('[<>]', '')),
+        'start': lambda rows: rows.locus.position,
+        'end': lambda rows: rows.info.END,
+        'sv_callset_Hemi': lambda rows: rows.info.N_HET,
```

Comment:
In the seqr UI we will show Hom and Hemi for specific populations but currently don't show Het for anything, so I thought mapping this to Hemi so we could show it would be okay. However, since it seems that referring to this as "Hemi" would be inaccurate we should change this field to `sv_callset_Het` as you propose and then update the seqr UI to show it